packages/web-components/fast-components/src/slider-label/slider-label.ts

```diff
@@ -31,6 +32,15 @@ export class SliderLabel extends FASTElement {
         orientation: SliderOrientation.horizontal,
     };
 
+    @attr({ attribute: "disabled", mode: "boolean" })
+    public disabled: boolean; // Map to proxy element
+    private disabledChanged(): void {
+        this.disabled
+            ? this.classList.add("disabled")
+            : this.classList.remove("disabled");
+        this.setAttribute("aria-disabled", this.disabled.toString());
```

Comment:
I'm not sure that you need the setAttribute call here. The binding you have on `template` should handle it.

Reply:
removed setAttribute